src/libraries/Microsoft.PowerFx.LanguageServerProtocol/LanguageServer/LanguageServer.cs

```diff
@@ -35,6 +35,11 @@ public class LanguageServer
     {
         private const char EOL = '\n';
 
+        /// <summary>
+        /// This const represents the dummy formula that is used to create an infrastructure needed to get the symbols for Nl2Fx operation.
+        /// </summary>
+        public const string Nl2FxDummyFormula = "\"Dummy Nl2Fx Check\"";
```

Comment:
For Nl2FX, we run scope.Check() multiple times. Initially, we run it on Dummy expression to get access to CheckResult which has necessary infrastructure to build the context (symbol tables). And then we run Check() on expressions returned from Nl2Fx model inside FinalCheck() function. Power Apps needs to distinguish between these Check calls as it doesn't use symbol tables but rather uses traditional name resolver. This change is not harmful yet gives deterministic way to distinguish between Check call being made on a dummy expression vs Check calls inside FinalCheck(). This change is not harmful because it doesn't matter what the dummy expression is. "1" as a dummy expression was too common.